src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -275,14 +175,31 @@
            construct-base-url
            utils/validate-cli-base-url!)))))
 
+(defn export!
+  [outfile nodes-data]
+  (with-open [tar-writer (archive/tarball-writer outfile)]
+    (let [metadata (export-metadata (now))]
+      (log/info (:msg metadata))
+      (utils/add-tar-entry tar-writer metadata))
+    (doseq [tar-item nodes-data]
+      (log/info (:msg tar-item))
```

Comment:
since we're putting stuff on the wishlist, it would be super cool to have counts of the exported objects in that final line if that would be simple to do.